.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release shared-workflows
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: googleapis/release-please-action@7987652d64b4581673a76e33ad5e98e3dd56832f # v4.1.3
+        id: release
+        with:
+          release-type: simple
+          skip-github-pull-request: true
```

Comment:
are you sure that we want to do this? to me, releasing on every push seems a bit much... why wouldn't people use the SHA then? open to arguments 🙂

Reply:
yeah was unsure about this as well, I firstly left it there so the development was easier, but maybe we need to enable pull requests and have open PRs (which will be updated on every push) until we decide when to merge them

Reply:
yeah I'd do that. From reading about that action it's the flow they want you to use... collect up the changes in a PR and then set that free when you're ready. Feels nice, and would be good to try it for a bit 🙂

(even if I think the thing about deleting/replacing tags is 💀... but that's what GitHub Actions wants you to do!)